docs/guides/update_versions.md

```diff
@@ -1,26 +1,25 @@
 ---
-title: Updating versions of Rocky
+title: Performing unsupported upgrades between Major Rocky Linux releases
 author: Steven Spencer
```

Comment:
The title isn't really correct now. I'm giving a procedure that is the supported way (using a clean install and a package list) of going from one version to another, so it isn't an "unsupported update" like using Elevate would be. The title you used is also very long, which means that the left-hand TOC will be cluttered by the length.